Codes/tests/DAv/test_defensers.py

```diff
@@ -94,3 +94,8 @@ def test_moving_borders(defensers_in_borders):
     defenser2.step(2)
     defenser2_new_pos_righ = defenser2.get_position()
     assert defenser2_new_pos_righ == defenser2_prev_pos
+
```

Comment:
Add a test to verify that after the attacker received +1 for killing a defenser it is then again zero when moving.

Reply:
Done in test_env.py with the method `test_killing_rewards`